framework/src/engine/consensus/consensus.ts

```diff
@@ -536,11 +576,32 @@ export class Consensus {
 
 		if (!options.skipBroadcast) {
 			this._network.send({ event: NETWORK_EVENT_POST_BLOCK, data: block.getBytes() });
+			this._logger.debug(
+				{
+					id: block.header.id,
+					height: block.header.height,
+					generator: block.header.generatorAddress.toString('hex'),
+					number_of_transactions: block.transactions.length,
+					number_of_assets: block.assets.getAll().length,
+				},
+				'Block broadcasted',
+			);
 			this.events.emit(CONSENSUS_EVENT_BLOCK_BROADCAST, {
 				block,
 			});
 		}
 		const events = await this._executeBlock(contextID, stateStore, block);
+		this._logger.info(
```

Comment:
I think this log should be at the end of this function since it can still fail at this point

Reply:
also property name should always be camel case